library/std/src/env.rs

```diff
@@ -472,8 +471,8 @@ pub struct JoinPathsError {
 /// # }
 /// ```
 ///
-/// Using `env::join_paths` with [`env::split_paths`] to append an item to the `PATH` environment
-/// variable:
+/// Using [`env::join_paths()`] with [`env::split_paths()`] to append an item to
```

Comment:
This is on the documentation for `join_paths`. No need to link to the current function ;)
```suggestion
/// Using `env::join_paths()` with [`env::split_paths()`] to append an item to
```
and please also remove the unused link on line 493.

Reply:
You forgot to remove the unused link.